ci/nightly/pipeline.yml

```diff
@@ -59,6 +59,7 @@ steps:
           - { value: persist-maelstrom-single-node }
           - { value: persist-maelstrom-multi-node }
           - { value: unused-deps }
+          - { value: sql-features }
```

Comment:
Can you please name the directory and the buildkite step it "sql-feature-flags" or even just `feature-flags`. This way there will be no confusion as to what exactly is being tested here.